modules/tinymce/src/plugins/table/main/ts/actions/ResizeHandler.ts

```diff
@@ -33,12 +33,43 @@ export const getResizeHandler = function (editor: Editor): ResizeHandler {
     return elm.nodeName === 'TABLE';
   };
 
-  const lazyResize = function () {
-    return resize;
-  };
+  const lazyResize = () => resize;
```

Comment:
That would not be functionally the same and would cause bugs, as atm it's lazily evaluating `resize`. Using `Fun.constant` would evaluate it once when the `lazyResize` variable is declared.